scripts/menu.gsc

```diff
@@ -536,10 +536,39 @@ MenuOptionsPlayer(menu, player)
             self addMenu(menu, "Upgraded Specials");
             self addOpt("Zitros Orbital Arbalest", &GiveClientWeapon, "launcher_standard_t8_upgraded", self);
             self addOpt("Thekrauss Refibrillator++", &GiveClientWeapon, "special_ballisticknife_t8_dw_upgraded", self);
-            if(BO4GetMap() == "Blood" || BO4GetMap() == "AO" || BO4GetMap() == "Tag" || BO4GetMap() == "Classified"){
-                self addOpt("Porters X2 Ray Gun", &GiveClientWeapon, "ray_gun_upgraded", self);
-                self addOpt("Porters Mark II Ray Gun", &GiveClientWeapon, "ray_gun_mk2_upgraded", self);
+            self addOpt("Porters X2 Ray Gun", &GiveClientWeapon, "ray_gun_upgraded", self);
+            self addOpt("Porters Mark II Ray Gun", &GiveClientWeapon, "ray_gun_mk2_upgraded", self);
```

Comment:
The Ray Gun and Mk2 Ray gun are Aether Only, so they need a check to ensure they only show up on Aether. This is Blood, Classified, Alpha, and Tag.

Reply:
Done that now